protocols/rendezvous/tests/rendezvous.rs

```diff
@@ -91,7 +110,8 @@ async fn given_successful_registration_then_refresh_ttl() {
 
     alice
         .behaviour_mut()
-        .register(namespace.clone(), roberts_peer_id, None);
+        .register(namespace.clone(), roberts_peer_id, None)
+        .expect("Should send register request");
```

Comment:
Nit: My preference here would be to use `unwrap` _for a function that should succeed as part of the test_.

If you wanted to use `expect`, the message should document the invariant, why it shouldn't fail. In this case that would be something like "we know our external address". However, that will get quite verbose hence my suggestion to just use `unwrap`.